Common/HttpWebRequestWrapperLib/HttpWebRequestWrapper.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Text.Json;
+
+namespace HttpWebRequestWrapperLib
+{
```

Comment:
I don`t now, how it got here, but we already have HttpWebRequestWrapper.
I think you should merge new develop into your branch and take en existing library from there.